File: CryBot.Contracts/ITrade.cs
namespace CryBot.Contracts
{
    public interface ITrade
    {
        bool IsActive { get; set; }
        
        ICryptoOrder BuyOrder { get; set; } 
        
        ICryptoOrder SellOrder { get; set; }
        
        decimal MaxPricePerUnit { get; set; }
    }
}

File: CryBot.Web/ClientApp/app/models/api/ITrade.ts
import { ICryptoOrder } from "./ICryptoOrder";
import { TradeStatus } from "../TradeStatus";

export interface ITrade {
    isActive: boolean;
    buyOrder: ICryptoOrder;
    sellOrder: ICryptoOrder;
    maxPricePerUnit: number;
    profit: number;
    status: TradeStatus;
}